content/reference/cli/pulumi_login.md

```diff
@@ -17,25 +17,25 @@ The service manages your stack's state reliably. Simply run
     $ pulumi login
 
 and this command will prompt you for an access token, including a way to launch your web browser to
-easily obtain one. You can script by using PULUMI_ACCESS_TOKEN environment variable.
+easily obtain one. You can script by using `PULUMI_ACCESS_TOKEN` environment variable.
```

Comment:
Just noticed that this is a change to a generated file. It'd be best to just revert the changes to this file (since they'll be clobbered over the next time we regenerate), and instead make any improvements to the source: https://github.com/pulumi/pulumi/blob/8e0da42412dd3140782446db874ab7a73ec5cda1/cmd/login.go#L40

Reply:
Great catch! This is being tracked in https://github.com/pulumi/pulumi/pull/2810. I'll go ahead and remove the changes to `pulumi login` from this PR